cob_default_robot_config/raw3-1/arm_joint_configurations.yaml

```diff
@@ -0,0 +1,11 @@
+joint_names: ['arm_shoulder_pan_joint', 'arm_shoulder_lift_joint', 'arm_elbow_joint','arm_wrist_1_joint', 'arm_wrist_2_joint', 'arm_wrist_3_joint']
```

Comment:
this file needs to be at `cob_default_robot_config/robots/raw3-1/script_server/arm_joint_configurations.yaml`
there is one available already at this location which seems to be identical: https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_default_robot_config/robots/raw3-1/script_server/arm_joint_configurations.yaml

thus, please remove this file from the PR